rust/nix-expr/src/eval_state.rs

```diff
@@ -131,6 +137,46 @@ impl EvalState {
         }
         self.get_string(value)
     }
+    pub fn realise_string(
```

Comment:
Is there a reason you call it `realise_string` rather than matching the raw API and calling it `string_realise`?

Reply:
Most of the methods in `EvalState` are verb-noun. For instance, it's consistent with `require_string`.
The C API uses a more "hierarchical" noun-verb scheme, but not consistently. I think this was chosen to somewhat simulate namespaces in C. In Rust we don't have to do that.